ingress-nginx-controller-1.12.yaml

```diff
@@ -498,6 +502,51 @@ pipeline:
         && setcap -v cap_net_bind_service=+ep ${{targets.destdir}}/usr/bin/nginx
 
 subpackages:
+  - name: ingress-nginx-controller-bitnami-compat-${{vars.nginx-ingress-major-minor}}
```

Comment:
The name is not `${{package.name}}-bitnami-compat` to be consistent with the name of the standard compat package.

Reply:
me & Woj put the x.y version at middle for cert-manager compats. (non-blocking, this works as well)

Reply:
I'm not sure I like it, but it makes more sense than having inconsistency between regular compat and bitnami compat. Whatever you decide to use - let's do it.

Reply:
I agree, I don't like it either, but I prefer consistency